apps/cms/scripts/seed.mjs

```diff
@@ -127,6 +127,13 @@ async function main(args) {
     languages
     CASCADE`)
 
+  await knex.raw(
+    `INSERT INTO directus_webhooks (
+      "name", "method", "url", "status", "data", "actions", "collections", "headers"
+    ) VALUES (
+      'CMS Cache Content', 'POST', 'http://localhost:3002/webhooks/directus', 'active', 't', 'create,update,delete', 'application,collections,countries,faqs,homepage,languages,nft_templates,pack_templates,page,rarities,sets', NULL
+    )`
+  )
```

Comment:
This definitely works for now, I think? Won't this try to hit an endpoint that doesn't exist until Scribe is merged in?

I guess a nice-to-have would be to instead use the Directus API during Scribe's startup to ensure a webhook is configured: https://docs.directus.io/reference/system/webhooks/

Reply:
Yeah I'll pull this out and we can have scribe set it up